pyproject.toml

```
@@ -89,9 +116,18 @@ skip-magic-trailing-comma = false
 # Like Black, automatically detect the appropriate line ending.
 line-ending = "auto"
 
+[tool.setuptools]
+package-dir = { "" = "src" }
+
 [tool.setuptools.packages.find]
-where = [
-    "src",
-]
+where = ["src"]
+
+[project.entry-points.'nomad.plugin']
+
+general_schema = "nomad_measurements.general:general_schema"
+xrd_schema = "nomad_measurements.xrd:xrd_schema"
+xrd_parser = "nomad_measurements.xrd.parser:xrd_parser"
+transmission_schema = "nomad_measurements.transmission:transmission_schema"
+transmission_parser = "nomad_measurements.transmission:transmission_parser"
```

As mentioned before, we can have general names for the entry point objects: schema and parser, to avoid repetition in the entry point path
```suggestion
general_schema = "nomad_measurements.general:schema"
xrd_schema = "nomad_measurements.xrd:schema"
xrd_parser = "nomad_measurements.xrd.parser:parser"
transmission_schema = "nomad_measurements.transmission:schema"
transmission_parser = "nomad_measurements.transmission:parser"
```